src/taxes/income-tax.ts

```diff
@@ -595,26 +622,19 @@ export function getTotalTaxAmount(
     yearsToInflate = 0,
 ): number {
     const provTax = getProvincialTaxAmount(provincialCode, grossIncome, inflationRate, yearsToInflate);
-    const provSurtax = getProvincialSurtaxAmount(provincialCode, provTax, inflationRate, yearsToInflate);
     const fedTax = getFederalTaxAmount(grossIncome, inflationRate, yearsToInflate);
-
-    return provTax + provSurtax + fedTax;
+    const abatement = getProvincialAbatement(provincialCode, fedTax);
+    return Math.max(provTax, 0) + Math.max(fedTax - abatement, 0);
 }
 
-export function calculateEffectiveTaxRate(income: number, province: ProvinceCode): number {
+export function calculateEffectiveTaxRate(
+    income: number,
+    province: ProvinceCode,
+    inflationRate = 0,
+    yearsToInflate = 0,
+): number {
     if (income <= 0) {
         return 0;
     }
```

Comment:
J'aime les opérations ternaire
```suggestion
        return income <= 0 ? 0 : (getTotalTaxAmount(province, income, inflationRate, yearsToInflate)) / income;
```

Reply:
Moi moins :P